src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -186,6 +186,21 @@ const mockVsCodeApi = () => ({
                       type: "backend"
                     }
                   },
+                  {
+                    name: "Moleculer",
+                    displayName: "Moleculer",
+                    summary: "JavaScript framework",
+                    author: "Various",
+                    order: "1",
+                    licenses:
+                      "[Moleculer](https://github.com/moleculerjs/moleculer/blob/master/LICENSE)",
+                    platforms: ["Web"],
+                    languages: ["Any"],
+                    tags: {
+                      version: "0.13.10",
+                      preview: false
```

Comment:
If you'd like to deploy a "beta" (preview) version of Moleculer first, you can set this one to `true`. Otherwise we will need to spend more time to make sure is ready for a prod release.

Reply:
Done https://github.com/microsoft/WebTemplateStudio/pull/1173/commits/5e962fa4222cdd7036a40ef014e5714bb1e8c246

Reply:
@jcoc611-microsoft let me know if you need anything else 😉